css/SettingsSection.scss

```diff
@@ -26,22 +26,57 @@
 	.settings-section {
 		display: flex;
 		align-items: center;
-		margin-top: 5px;
+	}
+}
+
+.new-addressbook-entry {
+	display: flex;
+	align-items: center;
+	.icon-add {
+		width: 44px;
+		height: 44px;
```

Comment:
Could be grouped with other icon styles in a shared class like `settings-line__icon` or something.

Reply:
done